x-pack/plugins/triggers_actions_ui/public/application/sections/field_browser/components/field_items/field_items.tsx

```diff
@@ -134,7 +126,7 @@ const getDefaultFieldTableColumns = ({
     width: '130px',
   };
 
-  return [nameColumn, ...(showDescriptionColumn ? [descriptionColumn] : []), categoryColumn];
+  return [nameColumn, ...[descriptionColumn], categoryColumn];
```

Comment:
Are there instances we don't want to show description?

Reply:
This is a great question that I was wondering if observability folks had feedback on this. Previously description was only available on the security solution because we loaded the data (+description) from the `indexFields` search strategy. Now, the description data is available in that ecs const so I can't foresee a place between the security solution fields browser and observability's that the description would not want to be shown. That `showDescriptionColumn` property was used by Observability because they didn't have access to the search strategy that the security solution uses to load the description data.

Reply:
@XavierM can you confirm whether `showDescriptionColumn` should be kept around or not?